Update hooks for pyzmq 22

Dereas opened the pull request:
import zmq.libzmq fails for pyzmq 22 on windows. The libzmq
dependencies are found in %ZMQ_MODULE_PATH%/../pyzmq.libs

Resolves #950

Dereas committed:
Update hooks for pyzmq 22

import zmq.libzmq fails for pyzmq 22 on windows. The libzmq
dependencies are found in %ZMQ_MODULE_PATH%/../pyzmq.libs

marcelotduarte commented on cx_Freeze/hooks.py:
```diff
@@ -1328,7 +1328,13 @@ def load_zmq(finder: ModuleFinder, module: Module) -> None:
-            pass  # No bundled libzmq library
+            # For pyzmq 22 the libzmq dependencies are located in site-packages/pyzmq.libs
+            libzmq_folder = "pyzmq.libs"
+            libs_path = os.path.abspath(os.path.join(module.path[0], os.pardir, libzmq_folder))
```
I think this is more pythonic:
libs_path = os.path.join(os.path.dirname(module.path[0]), libzmq_folder)

marcelotduarte commented on cx_Freeze/hooks.py:
```diff
@@ -1328,7 +1328,13 @@ def load_zmq(finder: ModuleFinder, module: Module) -> None:
+            libzmq_folder = "pyzmq.libs"
+            libs_path = os.path.abspath(os.path.join(module.path[0], os.pardir, libzmq_folder))
+            if os.path.exists(libs_path):
+                finder.IncludeFiles(libs_path, os.path.join('lib', libzmq_folder))
```
use "lib"

marcelotduarte commented on cx_Freeze/hooks.py:
```diff
@@ -1328,7 +1328,13 @@ def load_zmq(finder: ModuleFinder, module: Module) -> None:
+            if os.path.exists(libs_path):
+                finder.IncludeFiles(libs_path, os.path.join('lib', libzmq_folder))
+            else:
+                pass  # No bundled libzmq library
```
Move this comment before to if, changing the text accordingly to eliminate the unnecessary else and pass

Dereas committed:
Cleanup pyzmq hook

- Use less cumbersome path manipulation
- Remove unnecessary else: pass section
- Replace '' by ""

marcelotduarte commented:
Thanks!
